servo/events.py

```diff
@@ -534,24 +537,7 @@ def decorator(fn: EventCallable) -> EventCallable:
                     localns=handler_localns,
                 ),
                 name=name,
-                callable_description="before event handler"
-            )
-        elif preposition == Preposition.on:
-            servo.utilities.inspect.assert_equal_callable_descriptors(
-                servo.utilities.inspect.CallableDescriptor(
-                    signature=event.signature,
-                    module=event.module,
-                    globalns=event_globalns,
-                    localns=locals(),
-                ),
-                servo.utilities.inspect.CallableDescriptor(
-                    signature=handler_signature,
-                    module=handler_module,
-                    globalns=handler_globalns,
-                    localns=handler_localns,
-                ),
-                name=name,
-                callable_description="event handler"
```

Comment:
These validations only differ by 2 parameters (signature of the first CallableDescriptor, and callable_description). The validation was not removed, it was refactored to reduce the amount of boilerplate code (really all 3 validations could be collapsed into one call to assert_equal_callable_descriptors instead of 3 largely redundant ones).

How does this make validation less robust? Validation for handlers was not deleted...